cpu/cortexm_common/vectors_cortexm.c

```diff
@@ -246,6 +261,10 @@ __attribute__((used)) void hard_fault_handler(uint32_t* sp, uint32_t corrupted,
         printf("EXC_RET: 0x%08" PRIx32 "\n", exc_return);
         puts("Attempting to reconstruct state for debugging...");
         printf("In GDB:\n  set $pc=0x%lx\n  frame 0\n  bt\n", pc);
+        int stack_left = _stack_size_left(HARDFAULT_HANDLER_REQUIRED_STACK_SPACE);
+        if(stack_left < 0) {
+            printf("\nISR stack overflowed by %d bytes max.\n", (-1 * stack_left));
```

Comment:
Wait, isn't it the other way around? Now that the stack usage refering to the entry pointer is known and given that the stack pointer has probably advanced (at least the stack usage didn't decline), I can tell that **max.** n bytes overflowed.